### ISSUE TITLE
508-defect-1 ❗Launchblocker [COGNITION]: Locations Page - mental health clinics must have discernible text or be removed

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21073]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Angela or Trevor_

## User Story or Problem Statement

As a user who might need to make use of mental health services, I need accurate information about what Eastern OK locations have these services available, and links that can connect me with those services.

## Details

There are four (4) locations that have a Mental Health Clinic entry with empty `<a>` tags. These are problematic for two major reasons:

1. Veterans who might be in crisis need accurate information and links to that information. These empty entries slow down finding the right page with the right information.
2. These empty links are axe-core violations and will break the CI/CD build step when they are scanned for accessibility.
Screenshots attached below.

## Acceptance Criteria

- [ ] Empty mental health clinic entries are removed, OR
- [ ] A valid telephone number is added to each mental health clinic `<a>` tag
- [ ] Page passes an axe-core scan with 0 violations

## Environment
* https://staging.va.gov/eastern-oklahoma-health-care/locations/

## WCAG or Vendor Guidance (optional)
* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs
![Screen Shot 2021-03-05 at 4 08 56 PM](https://user-images.githubusercontent.com/934879/110180055-bcd8ec80-7dce-11eb-8d52-938b712b434a.png)

---
<img width="857" alt="Screen Shot 2021-03-05 at 4 21 03 PM" src="https://user-images.githubusercontent.com/934879/110180208-f873b680-7dce-11eb-8b84-9e12dd5f9fca.png">

**Comment by davidconlon** [comment]
@mmiddaugh we need to do a check for the mental health numbers and we should also understand why there would be a case where a mental health number might not be in lighthouse.
(This could be a data problem)

**Comment by davidconlon** [comment]
Following up on this:
Not all Clinics currently have a mental health number.
Yale Avenue VA Clinic is actually a Dental Clinic.

We may need to check with lighthouse on how these numbers are maintained because I seem to recall a desire to make these maintained in Drupal at some point.

**Comment by mmiddaugh** [comment]
Mental Health numbers are pulled from the API. 
1st Facilities FE task: Check to see that number is not null.

**Comment by mmiddaugh** [comment]
Issue is still present on https://www.va.gov/eastern-oklahoma-health-care/locations/mccurtain-county-va-clinic/

<details>
<summary Screenshots </summary>

![image](https://user-images.githubusercontent.com/55411834/112066609-4c4f0100-8b2c-11eb-8d2d-c25f3e46f111.png)

![image](https://user-images.githubusercontent.com/55411834/112067186-460d5480-8b2d-11eb-8af3-9219affc5010.png)

</details>

**Comment by mmiddaugh** [comment]
I believe this one is a duplicate for https://github.com/department-of-veterans-affairs/va.gov-team/issues/21791

**Comment by 1Copenut** [comment]
@mmiddaugh These may be duplicates, but it's possible they're peers. This ticket is specific to VAMCs' `/location` URL and template. #21791 is a collection of all the links missing discernible text that might span across one or more templates.

**Comment by mmiddaugh** [comment]
A fix (#21073) to resolve this issue has been merged and is available to be validated on Prod @1Copenut
